ladybug_comfort/apparent.py

```diff
@@ -0,0 +1,96 @@
+# coding=utf-8
+"""Utility functions for calculating the Apparent Temperature (AT)."""
+from __future__ import division
+
+import math
+
+
+def apparent_temperature(ta, rh, ws):
+    """Calculate apparent temperature (AT) from air temperature, relative humidity, and wind speed.
+
+    The Australian Apparent Temperature (AT) is a type of heat index that was used by the
+    Australian Bureau of Meteorology(ABM). It is based on a mathematical model published
+    by Robert G. Steadman in 1994 [1]. Two forms of the model are available, one including
+    radiation and one without. This algorithm uses the non-radiation version. [2]
+
+    Note:
+        [1] Steadman, R. G. (1994). Norms of apparent temperature in Australia. Aust. Met. Mag, 43, 1-16.
```

Comment:
It looks like you might be exceeding our [max character limit of 89 characters-per-line](https://github.com/ladybug-tools/contributing/wiki/python-style-guide#maximum-line-length) here. It might be helpful to add some rulers into the text editor that you are using to write this code.